test/index.test.js

```diff
@@ -205,11 +208,11 @@ describe('index', () => {
                         name: testBuildId,
                         container: testContainer,
                         launchVersion: testLaunchVersion,
-                        logVersion: testLogVersion,
                         serviceAccount: testServiceAccount
                     },
                     command: [
-                        `/opt/screwdriver/launch ${testApiUri} ${testToken} ${testBuildId}`
+                        `/opt/screwdriver/launch ${testApiUri} ${testStoreUri} `
```

Comment:
this duplicates the template in the code under test. Maybe use the actual values here instead?